src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/_help.py

```diff
@@ -468,6 +468,15 @@
     short-summary: Open a web app in a browser.
 """
 
+helps['webapp new'] = """
```

Comment:
There is no precedent for a "new" command in the CLI.  This should definitely be called something different.

Reply:
yes we will change this once we decide on a verb that also is CLI approved